_pages/en/strategy-open-first.md

```diff
@@ -0,0 +1,445 @@
+---
+layout: default
+title: "Open First Strategy"
+ref: open-first-strategy
+lang: en
+status: In Progress;primary
+sections: Strategies
+permalink: /open-first-strategy.html
+---
+<!-- the below cSpell statement says to ignore any text between HTML tags. E.g. it will ignore "th rowspan='2'" in this string: <th rowspan='2'> -->
+<!-- cSpell:ignoreRegExp /\<[^\>]+\>/ -->
+
+1. Introduction
+2. Vision
+3. Commitment
+4. Goals and outcomes
+5. Governing principles
+6. Implementing the strategy
+
+## 1. Introduction
+
+The Government of Canada (GC) aims to invigorate Canada's unique social market economy, promote competition and encourage SMEs — our innovators and entrepreneurs.
+We want to bring Canada's people together in an inclusive, open approach, to find new opportunities and transition to an inclusive, better digital environment that is ready for the realities of today's global economy.
+
+In all of this, open standards and open source software (OSS) have a role to play.
+
+OSS is present everywhere.
+All around the world, companies and public services are using open collaborative methods to innovate and build new solutions.
+It powers the cloud and provides professional tools for big data and for information and knowledge management.
+It is in supercomputers, blockchain, the internet of things and artificial intelligence.
+It is in the internet.
+It is in our phones and our TVs.
+It provides us with streaming media.
+It is in our cars.
+The chances are that, in any new project involving software, from kitchen appliances, through web-based public services to highly specialised industrial tools, most of the code will be based on OSS.
+
+Open standards are a set of rules designed to do a specific job in technology.
+Open standards refer to file formats, protocols and application interfaces that can be implemented by everyone (in OSS and proprietary software alike) since the specifications are available at no cost, and since their development and standardization is open and transparent.
```

Comment:
Open standards are the guidelines that enable international communities to speak a common language to build products (OSS and proprietary) in a structured fashion.  They are available at no cost since standardization is open and transparent.